android/src/main/kotlin/com/significo/recco/ReccoPlugin.kt

```diff
@@ -0,0 +1,112 @@
+package com.significo.recco
+
+import android.content.Context
+import com.recco.api.ui.ReccoApiUI
+import io.flutter.embedding.engine.plugins.FlutterPlugin
+import io.flutter.embedding.engine.plugins.activity.ActivityAware
+import io.flutter.embedding.engine.plugins.activity.ActivityPluginBinding
+import io.flutter.plugin.common.MethodCall
+import io.flutter.plugin.common.MethodChannel
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler
+import io.flutter.plugin.common.MethodChannel.Result
+
+/**
+ * ReccoPlugin
+ */
+class ReccoPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
+    private lateinit var channel: MethodChannel
+    private var context: Context? = null
+
+    /**
+     * FlutterPlugin
+     */
+    override fun onAttachedToEngine(flutterPluginBinding: FlutterPlugin.FlutterPluginBinding) {
+        channel = MethodChannel(flutterPluginBinding.binaryMessenger, "recco")
```

Comment:
maybe we can also extract `"recco` literal into a companion val, in the same way we have the other ones